fastfold/model/nn/primitives.py

```diff
@@ -29,6 +29,7 @@
     _chunk_slice,
 )
 
+from fastfold.habana.fastnn.custom_op import fused_softmax, fused_softmax_bias
```

Comment:
This line may break on non-habana environment, maybe need `habana.is_habana()`

Reply:
will fix in updated patch